_posts/0000-01-02-Dolat-Hamza.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
“If you think it’s expensive to hire a professional, wait until you hire an amateur.”
—Red Adair